### ISSUE TITLE
有没有计划支持AEC

### ISSUE THREAD

**Issue by hfwanguanghui** [Xiaobin-Rong/gtcrn#39]
1.非常好的项目，请问作者有没有计划支持GTCRN的回声消除模型。

**Comment by Xiaobin-Rong** [comment]
你好，目前的计划是进一步优化该模型在SE中的性能，暂无拓展到AEC的计划~

**Comment by Xiaobin-Rong** [comment]
@TungyuYoung Yes, it's a great direction. I have tried vanilla knowledge distillation but improved little. So you have done it successfully?

**Comment by ercandogu-elevear** [comment]
@Xiaobin-Rong what kind of teacher model have you tried? I tried to expand the GTCRN model changing the channels and so on but the teacher model doesn't converge really well.

**Comment by Xiaobin-Rong** [comment]
@ercandogu-elevear Yes, I tried expanding GTCRN as teacher models. I also found similar phenomena of bad convergence when expanding GTCRN to a certain scale, for example, 6 GMacs. Actually, the phenomenon keeps the same when I change GTCRN to MTFAA. But if we modify the models from causal ones to non-causal ones, the convergence becomes normal.

**Comment by ercandogu-elevear** [comment]
@Xiaobin-Rong Is there any possibility you can share the teacher model? I am really interested in KD area, trying in for the first time,  but I couldn't create a teacher model that converges well.  Therefore I don't get any improvements.
